src/content/guides/how-to-use-a-wallet/index.md

```diff
@@ -18,7 +18,7 @@ Do you want to receive crypto into your wallet?
 
 Each Ethereum account has its own receiving address which is a unique sequence of numbers and letters. The address functions like a bank account number. Ethereum addresses will always start with “0x”. You can share this address with anyone: it is safe to do so.
 
-Your address (which is sometimes called a "public key") is like your home address: you need to tell people what it is so they can find you. It is safe to do this, because you can still lock your front door with another key only you control so that no-one can get in, even if they know where you live.
+Your address (which is derived from your "public key") is like your home address: you need to tell people what it is so they can find you. It is safe to do this, because you can still lock your front door with another key only you control so that no-one can get in, even if they know where you live.
```

Comment:
If we aren't leaning into this misunderstanding, we can just remove this.

```suggestion
Your address is like your home address: you need to tell people what it is so they can find you. It is safe to do this, because you can still lock your front door with another key only you control so that no-one can get in, even if they know where you live.
```